components/storage/blockdevice/COMPONENT_SPIF/SPIFBlockDevice.cpp

```diff
@@ -470,9 +470,6 @@ spif_bd_error SPIFBlockDevice::_spi_send_read_command(int read_inst, uint8_t *bu
     uint32_t dummy_bytes = _dummy_and_mode_cycles / 8;
     int dummy_byte = 0;
 
-    // csel must go low for the entire command (Inst, Address and Data)
-    _cs = 0;
```

Comment:
Shouldn't this be replaced with `_spi.select()`?
This also applies to the lines below...
Maybe there is a good reason, but I don't understand it?

Reply:
this is write function:
int SPI::write(int value)
{
    select();
    int ret = spi_master_write(&_peripheral->spi, value);
    deselect();
    return ret;
}
so there is done select and deselect.
additionaly lock and unlock is made in select and unselect.

Reply:
True about select and deselect, but looking at the comment `csel must go low for the entire command (Inst, Address and Data)` flipping the chip select off and on in every write function actually changes the behaviour of the whole function here. Chip select isn't low for the entire command now. Was that intentional?

Reply:
If i correct understand SPI is only interface between block device, so when we want write/ready to block device something we select cs and do operations on spi (one block device operation can take few spi actions). Function _

int DataFlashBlockDevice::read(void *buffer, bd_addr_t addr, bd_size_t size)

_ make few that operation and i cannot see reason to all time hold down cs (in my opinion cs should be down only when write spi operation).
@SeppoTakalo please review my understanding.